### ISSUE TITLE
html-webpack-plugin output html , interior body empty

### ISSUE THREAD

**Issue by MrZhang123** [jantimon/html-webpack-plugin#622]
use html-webpack-plugin output html page , but in output page lost body inner html element , like `<div id="app"></div>` , here is my input output and webpack.config file :

## input 

```html
    <!DOCTYPE html>
    <html lang="en">
    <head>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=0, minimum-scale=1.0, maximum-scale=1.0">
        <link rel="icon" href="static/pic/logo.png" type="image/x-icon">
        <link rel="stylesheet" href="static/weui/weui.min.css">
        <link rel="stylesheet" href="static/index.css">
        <title>APP</title>
    </head>
    <body>
        <div id="app"></div>
        <script src="./dist/vendor.js"></script>
        <script src="./dist/app.js"></script>
    </body>
    </html>
```

## output 

```html
    <!DOCTYPE html>
    <html>
      <head>
        <meta charset="UTF-8">
        <title>SPA</title>
      <link href="/dist/style.df241090c6a9e0a6bf26.css" rel="stylesheet"></head>
      <body>
      <script type="text/javascript" src="/dist/vendor.df241090c6a9e0a6bf26.js"></script><script type="text/javascript" src="/dist/app.df241090c6a9e0a6bf26.js"></script></body>
    </html>
```

## part webpack.config.js 

```js
    const htmlWebpackPlugin = require('html-webpack-plugin');
    
        plugins: [
            new htmlWebpackPlugin({
                title:'SPA',
                filename: 'assets/index.html'
            })
        ],
```

**Comment by Squidies** [comment]
I believe you have to specify the `template` option to your index file (or template file if using a templating engine) otherwise htmlwebpackplugin will default to its own.

**Comment by MrZhang123** [comment]
@Squidies yes , i solve it , thanks

**Comment by MrZhang123** [comment]
fixed #622
